__docs__/wonder-blocks-labeled-field/labeled-field.stories.tsx

```diff
@@ -169,3 +170,140 @@ export const Light: StoryComponentType = {
+                        error={singleSelectError}
+                        field={
+                            <SingleSelect
+                                placeholder="Choose a fruit"
+                                selectedValue={singleSelectValue}
+                                onChange={(value) => {
+                                    setSingleSelectValue(value);
+                                    if (value === "strawberry") {
+                                        setSingleSelectError(
+                                            "Don't select strawberry.",
+                                        );
+                                    } else {
+                                        setSingleSelectError("");
+                                    }
+                                }}
+                                error={!!singleSelectError} // should not need to do this! LabeledField should set this
+                            >
+                                <OptionItem label="Mango" value="mango" />
+                                <OptionItem
+                                    label="Strawberry"
+                                    value="strawberry"
+                                />
+                                <OptionItem label="Banana" value="banana" />
+                            </SingleSelect>
+                        }
+                    />
+
+                    <LabeledField
+                        {...args}
+                        label="Fruits"
+                        error={multiSelectError}
+                        field={
+                            <MultiSelect
+                                selectedValues={multiSelectValue}
+                                onChange={(value) => {
+                                    setMultiSelectValue(value);
+                                    if (value.length < 2) {
+                                        setMultiSelectError(
+                                            "Select at least 2",
+                                        );
+                                    } else {
+                                        setMultiSelectError("");
+                                    }
+                                }}
+                                error={!!multiSelectError}
+                            >
+                                <OptionItem label="Mango" value="mango" />
+                                <OptionItem
+                                    label="Strawberry"
+                                    value="strawberry"
+                                />
+                                <OptionItem label="Banana" value="banana" />
+                            </MultiSelect>
+                        }
+                    />
+                    <Button
+                        type="submit"
+                        onClick={() => {
+                            // Example where everything has an error
```

Comment:
error message for textarea and textfield will be shown because of LabeledField, however, error styling is not applied to the field element because they don't currently support an error prop (just built-in validation)